### PR TITLE
fix: Replacing media breakpoint static CSS to dynamic for WooCommerce

### PR DESCRIPTION
**Pull request by imnavanath**
- Removing breakpoint's CSS from static to dynamic

### CODE REVIEW

File: inc/compatibility/woocommerce/class-astra-woocommerce.php [hunk 1]
```diff
@@ -960,6 +1059,60 @@ public function add_styles() {
...
+				'.woocommerce.mobile-columns-1 ul.products li.product:nth-child(n), .woocommerce-page.mobile-columns-1 ul.products li.product:nth-child(n)' => array(
+					'margin-right' => '0',
+				),
+				'.woocommerce #content table.cart .product-thumbnail' => array(
+					'display'    => 'block',
+					'text-align' => 'center !important',
+				),
+				'.woocommerce #content div.product .woocommerce-tabs ul.tabs li, .woocommerce-page #content div.product .woocommerce-tabs ul.tabs li' => array(
+					'display'     => 'block',
+					'margin-righ' => '0',
```
Comment (Balachandark) [inline inc/compatibility/woocommerce/class-astra-woocommerce.php]: right spelling is incorrect

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -604,6 +604,13 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 				'.ast-separate-container .ast-article-post, .ast-separate-container .ast-article-single' => array(
 					'padding' => '1.5em 2.14em',
 				),
+				'.ast-separate-container #content .ast-container' => array(
+					'padding-left'  => '0.54em',
+					'padding-right' => '0.54em',
+				),
+				'.ast-separate-container #secondary'    => array(
+					'padding-top' => '0',
```
Comment (Balachandark) [inline inc/class-astra-dynamic-css.php]: 0 doesn't have to be a string

File: inc/compatibility/woocommerce/class-astra-woocommerce.php (hunk 1 above)
Comment (imnavanath) [reply inc/compatibility/woocommerce/class-astra-woocommerce.php]: Corrected spelled now

File: inc/class-astra-dynamic-css.php (hunk 2 above)
Comment (imnavanath) [reply inc/class-astra-dynamic-css.php]: Remove quotes from 0

**Pull request merged**